app/reducers/items.js

```diff
@@ -60,6 +60,10 @@ export const save = (state = [], action) => {
   switch(action.type) {
     case 'ADD_SAVE':
     return [...state, action.saved]
+    case 'DELETE_SAVED':
+    return state.filter((savedRecipe)=> {
```

Comment:
I would make this an implicit return.
`return state.filter((savedRecipe)=> savedRecipe !== action.savedRecipe)`